changelog.md

```diff
@@ -1,3 +1,13 @@
+## 3.4.2 / 2021-01-07
+
+### Fixed Issues
+
+- UX: [4023](https://github.com/tracim/tracim/issues/4023)
```

Comment:
Is this an issue? Or rather a new functionality?

Reply:
good question

Reply:
it's a feature

Reply:
thanks, I have not really read isse label :(